session/auth/authprovider/authprovider.go

```diff
@@ -18,26 +21,28 @@ import (
 	"github.com/docker/cli/cli/config"
 	"github.com/docker/cli/cli/config/configfile"
 	"github.com/docker/cli/cli/config/types"
-	"github.com/moby/buildkit/session"
-	"github.com/moby/buildkit/session/auth"
-	"github.com/moby/buildkit/util/progress/progresswriter"
 	"github.com/pkg/errors"
 	"golang.org/x/crypto/nacl/sign"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
+
```

Comment:
Wonder why linter didn't catch this empty line.

Reply:
https://stackoverflow.com/questions/38704769/golang-import-grouping-by-package

Reply:
Ok, but we don't use this for any other files so this looks out of place.